catalogs/qhimm.xml

```diff
@@ -2222,4 +2222,37 @@ Everyone has a Curse Ring Equipped, the challenge is to win before Death Sentanc
     </Mod>
 
   </Mods>
+
+## WereXIII - by Wolfman
+    <Mod>
+      <ID>1iwfYEec-eu6K4hB6zD06dok1jH92lX3n</ID>
+      <Author>Tsunamods</Author>
+      <Link>https://drive.google.com/file/d/1iwfYEec-eu6K4hB6zD06dok1jH92lX3n/view?usp=drive_link</Link>
+      <LatestVersion>
+        <Link></Link>
```

Comment:
Here instead the download link should be put, but instead with this syntax: `iros://GDrive/ID` where ID is the unique UUID drive gives you after `/d/`